src/routes/Dashboard/DropDown.jsx

```diff
@@ -26,7 +26,7 @@ export default function DropDown({setShowProfile, dropDownRef, className}) {
             <div className="w-full">
                 {
                     links.map(link =>
-                        <Link to={link.href}
+                        <Link to={link.href} key={Math.random()}
```

Comment:
This is a very bad practice, you can use the same `href` as the `key` if you know all the `href` are different